plugins/chat/spec/fabricators/chat_fabricator.rb

```diff
@@ -63,10 +63,10 @@
 Fabricator(:chat_message_without_service, class_name: "Chat::Message") do
   user
   chat_channel
-  message { Faker::Lorem.paragraph_by_chars(number: 500).gsub("...", "…").gsub("..", "…") }
+  message { Faker::Lorem.words(number: 5).join(" ") }
```

Comment:
While working on all the specs I noticed more cases which were causing failures (for reasons similar we had `.gsub("...", "…").gsub("..", "…")`

I figure it would be better to have simple words to limit this kind of issues.